pkg/labels/labels_test.go

```diff
@@ -485,3 +486,39 @@ func TestLabels_Has(t *testing.T) {
 		testutil.Equals(t, test.expected, got, "unexpected comparison result for test case %d", i)
 	}
 }
+
+func BenchmarkLabelsString(b *testing.B) {
```

Comment:
this benchmark doesn't need to stay but it was useful for optimizing the `String` function so far